cookiecutter.json

```diff
@@ -5,6 +5,7 @@
     "description": "A short description of the project.",
     "open_source_license": ["MIT", "BSD-3-Clause", "No license file"],
     "s3_bucket": "[OPTIONAL] your-bucket-for-syncing-data (do not include 's3://')",
+    "s3_prefix": "[OPTIONAL] prefix of your data in S3",
```

Comment:
This is not descriptive enough - can you expand so that people know what a prefix is and what a reasonable choice might me?

Reply:
Please add a default prefix of `data/`.

Reply:
I wanted to add a default of `data/` but didn't see an option to. Would this just be in the Makefile if there is no prefix set it to `data/`